wordpress-mobile/gutenberg-mobile
# Add UI tests to unsupported block editor

Addresses: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2358

Adds UI tests to the unsupported block editor.

Test that tapping on the help button of an unsupported block shows the bottom sheet. Tapping on the edit button in the bottom sheet shows the unsupported block web view editor.

| Android | iOS |
| -- | -- |
| <img src="https://user-images.githubusercontent.com/1898325/83586559-cae59e00-a51a-11ea-82cf-b7f200699014.gif"> | <img src="https://user-images.githubusercontent.com/1898325/83586573-d769f680-a51a-11ea-8ae5-a6616b3b5d40.gif"> |



To test:
1. Ensure tests are green (if running locally, run `yarn test:e2e:ios:local __device-tests__/gutenberg-editor-unsupported-blocks.test.js`)
2. Optional: Ensure the test catches a failure to open the web view; comment out these lines and ensure the tests fail: https://github.com/WordPress/gutenberg/pull/21150/files#diff-48d07c563702293f97bb8137c7ad9057R115-R119

PR submission checklist:
